Chef/AddRelativityVersionToSolutionSnapshotDatabase.ps1

```diff
@@ -93,7 +93,55 @@ $global:salesforceSessionObject = $null
 
 [string] $workspaceFolderGroupSecurityGuid = '07B44E66-1DDF-4607-987F-150C9C772B43'
 [string] $workspaceFolderGroupSecurityVersion = '1.0.0.0'
- 
+
+[string] $solutionSnapshotGuid = 'D51C3E3E-EBF4-402A-B41E-35C4018C8396'
+[string] $solutionSnapshotVersion = '4.0.0.4'
+
+class Application {
+  [string]$Name
+  [string]$Guid
+  [string]$Version
+
+  Application($name, $guid, $version) {
+    $this.Name = $name
+    $this.Guid = $guid
+    $this.Version = $version
+  }
+}
+
+[Application[]] $applications = @(
+  [Application]::new('Propogate Coding Post-Import', $propogateCodingPostImportGuid, $propogateCodingPostImportVersion1),
+  [Application]::new('Propagate Coding Post-Import', $propogateCodingPostImportGuid, $propogateCodingPostImportVersion2),
```

Comment:
can we combine propogateCodingPostImportVersion1 and propogateCodingPostImportVersion2 into one input. may be have a semi colon separated value so that we can specify multiple versions for the same application.